demos/k8s-demo/chart/demo-app-secretless/README.md

```diff
@@ -0,0 +1,152 @@
+# demo-app-secretless
+
+[The Secretless Broker](https://github.com/cyberark/secretless-broker) is a connection broker which relieves client applications of the need to directly handle secrets to target services such as databases, web services, SSH connections, or any other TCP-based service.
+
+## TL;DR;
+
+```bash
+$ helm install -f values.yaml .
+```
+
+## Introduction
+
+This chart bootstraps a deployment of a configurable demo-application, which uses [PostgreSQL](https://github.com/docker-library/postgres) as its backend, with Secretless on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. Secretless seamlessly manages the database connection for this application.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release \
+ --set applicationDBAddress="${DB_URL}" \
+ --set applicationDBUsername="${DB_USER}" \
+ --set applicationDBPassword="${DB_INITIAL_PASSWORD}"
```

Comment:
Is this really how we'd recommend a customer deploy an app that needed Secretless - with the DB user and password in the `helm install` command? I don't believe that's true - this is a red flag to me.

Reply:
Based on the recommended flow below - 
1. set env
2. install pg
3. configure database
4. deploy apps

I would expect the step where you store the secret values to occur in #2 or #3. deploying the app with helm should just include 1. loading the secretless config, 2. deploying the app and the sidecar

We want the flow here to be consistent with the flow we recommend elsewhere - that there is an "admin" path and a "non-admin" path. The admin deploys and configures pg and stores the secrets. Then the non-admin user deploys the app and the configured secretless sidecar.